src/main/java/frc/robot/Robot.java

```diff
@@ -291,37 +293,43 @@ private void spinControlPanel() {
         }
       }
 
-      String colorString;
+      String detectedColorString;
       Color detectedColor = colorSensor.getColor();
       ColorMatchResult match = colorMatcher.matchClosestColor(detectedColor);
       if (match.color == kBlueTarget) {
-        colorString = "Blue";
+        detectedColorString = "Blue";
       } else if (match.color == kRedTarget) {
-        colorString = "Red";
+        detectedColorString = "Red";
       } else if (match.color == kGreenTarget) {
-        colorString = "Green";
+        detectedColorString = "Green";
       } else if (match.color == kYellowTarget) {
-        colorString = "Yellow";
+        detectedColorString = "Yellow";
       } else {
-        colorString = "Unknown";
+        detectedColorString = "Unknown";
       }
-      detectedColorEntry.setString(colorString);
+      String lastDetectedColorString = detectedColorEntry.toString(); 
+      detectedColorEntry.setString(detectedColorString);
       confidenceEntry.setDouble(match.confidence);
-
+      
```

Comment:
Unneeded whitespace. A formatter should get rid of this.

Reply:
The formatter did not remove it, but I did.